shared/src/ledger/masp.rs

```diff
@@ -994,7 +998,7 @@ impl<U: ShieldedUtils> ShieldedContext<U> {
             return;
         }
         // If conversion if possible, accumulate the exchanged amount
-        let conv: Amount = conv.into();
+        let conv: I128Sum = FromNt(conv.into()).into();
```

Comment:
I found the name `FromNt` confusing here and had to go look it up. I left a comment about maybe using a different name.